src/libs/API/parameters/WorkspaceApprovalParams.ts

```diff
@@ -1,9 +1,7 @@
-import type {PolicyEmployee} from '@src/types/onyx';
-
 type CreateWorkspaceApprovalParams = {
     authToken: string;
     policyID: string;
-    employees: PolicyEmployee[];
+    employees: string;
```

Comment:
Should we add a comment about the `employees` prop structure here?

We already have some e.g. in ApproveMoneyRequestParams

```
/**
     * Stringified JSON object with type of following structure:
     * Array<{
     *   optimisticReportActionID: string;
     *   oldReportActionID: string;
     * }>
     */
```

Reply:
Awesome idea!